packages/engine/Source/Scene/VoxelTraversal.js

```diff
@@ -492,42 +498,23 @@ function loadAndUnload(that, frameState) {
   const frameNumber = that._frameNumber;
   const primitive = that._primitive;
   const shape = primitive._shape;
-  const { dimensions } = primitive;
   const targetScreenSpaceError = primitive.screenSpaceError;
   const priorityQueue = that._priorityQueue;
-  const keyframeLocation = that._keyframeLocation;
   const keyframeCount = that._keyframeCount;
-  const rootNode = that.rootNode;
+
+  // Some values are constant because of incomplete support for time-dynamic data
```

Comment:
I'm a bit confused by this comment. Which values, or types of values, can be time dynamic?

Reply:
On second look, my comment was incorrect. I noticed these values were always being assigned the same value in `addToQueueRecursive`, so I pulled them out as constants. But this is actually unrelated to support for time dynamic data.

As for what can be time dynamic, here is a brief explanation:
`keyframes` represent (integer) points along a time series where we have voxel samples in a time-varying dataset. `that._keyframeLocation` is the current time we want to render, which could be non-integer, requiring us to interpolate between samples in the time direction.

However, we currently don't allow any more than one `keyframe` (see [Cesium3DTilesVoxelProvider.js#L427C1-L430C4](https://github.com/CesiumGS/cesium/blob/50468896ca7d6071e0f8fb359598c03879fbf0a2/packages/engine/Source/Scene/Cesium3DTilesVoxelProvider.js#L427C1-L430C4)), so time-varying data cannot be loaded and rendered.

Reply:
There could be other providers that allow for multiple keyframes though. Not sure if that's relevant for this PR, but just something to keep in mind.